src/main/java/Main.java

```diff
@@ -1,39 +1,39 @@
-import lotto.model.LottoPurchaseTickets;
 import lotto.model.LottoStore;
+import lotto.model.LottoTicket;
+import lotto.model.LottoTickets;
 import lotto.model.WinningLottoTicket;
 import lotto.model.wrapper.LottoNumber;
+import lotto.model.wrapper.ManualLottoCount;
 import lotto.model.wrapper.Payment;
+import lotto.view.InputView;
 
-import java.util.Set;
-import java.util.stream.Collectors;
-
-import static lotto.utils.LottoUtil.convertTo;
-import static lotto.view.InputView.*;
 import static lotto.view.OutputView.printLottoResults;
 import static lotto.view.OutputView.printLottoTickets;
 
 public class Main {
 
     public static void main(String[] args) {
-        Payment payment = Payment.of(inputPayment());
-        LottoPurchaseTickets lottoPurchaseTickets = LottoStore.sell(payment);
+        Payment payment = InputView.inputData(InputView::inputPayment);
+
+        ManualLottoCount manualLottoCount = InputView.inputData(InputView::inputManualLottoCount);
+
+        LottoTickets manualLottoTickets = InputView.inputData(manualLottoCount, InputView::inputManualLotto);
+
+        LottoTickets lottoTickets = LottoStore.sell(payment, manualLottoTickets);
 
-        printLottoTickets(lottoPurchaseTickets);
+        printLottoTickets(lottoTickets, manualLottoTickets.size());
 
         WinningLottoTicket winningLottoTicket = inputWinningLottoTicket();
 
-        printLottoResults(lottoPurchaseTickets.checkAll(winningLottoTicket));
+        printLottoResults(lottoTickets.checkAll(winningLottoTicket));
```

Comment:
이왕 만든 `LottoTotalResult`를 사용하면 어떨까요?

```suggestion
        printLottoResults(LottoTotalResult.newInstance(lottoTickets.checkAll(winningLottoTicket)));
```